cmd/bundle.go

```diff
@@ -208,17 +207,15 @@ func runBundleNew(input *bundleNew) {
 		log.Fatal(err)
 	}
 
-	var artifacts []string
+	artifactDefinitions := map[string]map[string]interface{}{}
```

Comment:
We need both the name and schema now, so we'll use a map. This also means we'll sort it later.